src/main/java/com/linagora/james/mailets/json/ClassificationRequestBody.java

```diff
@@ -82,6 +84,7 @@ private static int getSanitizedMailSize(Mail mail) throws MessagingException {
     private static Optional<String> mainTextContent(MessageContent messageContent) {
         return messageContent.getHtmlBody()
             .filter(s -> !Strings.isNullOrEmpty(s))
+            .map(jsoupHtmlTextExtractor::toPlainText)
```

Comment:
Don't we do that also on james part already for indexation?
In the future we should find a way make that accessible on mailet so we don't do that multiple time.